src/core/namespace.c

```diff
@@ -1732,6 +1755,17 @@ int setup_namespace(
                         };
                 }
 
+                /* Will be used to add bind mounts at runtime */
+                if (setup_propagate)
+                        *(m++) = (MountEntry) {
+                                .source_const = strjoina(propagate_dir, "/", unit_id),
```

Comment:
I'm not sure if this strjoina is allowed.

Reply:
removed